package.json

```diff
@@ -49,7 +49,11 @@
       "hardenedRuntime": true,
       "entitlements": "assets/entitlements.mac.plist",
       "entitlementsInherit": "assets/entitlements.mac.plist",
-      "gatekeeperAssess": false
+      "gatekeeperAssess": false,
+      "extendInfo": {
+        "NSMicrophoneUsageDescription": "We need access to your microphone so people you talk to in a SugarTalk can hear you.",
+        "NSCameraUsageDescription": "Allow your conversational partners to see you in a SugarTalk. You can turn off your video anytime during a call."
```

Comment:
SugarTalk would like to access your camera so that people can see each other during the meeting